Submit new comment

kensayo opened the pull request:
- Create comments given an ID

kensayo committed:
Submit new comment

kensayo committed:
Fix linter errors

lfmnovaes commented on src/displayComments.js:
```diff
@@ -1,11 +1,22 @@
+const removeComments = () => {
```
I think it would be best to not remove all comments here. You can just simply add one new `<li>` element to the list and add it to the array of comments too(to use later on).

lfmnovaes reviewed (changes requested):
### Great work on implementing these project requirements @kensayo 💪 
It would be nice to consider doing what I've suggested below.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

kensayo committed:
Stop calling API to update comment list

- Remove the calling for API after a new comment is created
- Add into DOM the new comment

kensayo committed:
Merge branch 'dev' into leaveComment

kensayo committed:
Fix linter on comment

kensayo committed:
Fix error when no comments

kensayo committed:
Fixed linters
